actionkit/api/user.py

```diff
@@ -44,15 +44,28 @@ def create_user(self, user_dict):
         return rv
 
     def get_user(self, user_id):
+        print('get user called') #this is not printing. Is it not calling this at all?
         if getattr(self.settings, 'AK_TEST', False):
-            return TEST_DATA.get(user_id)
+            print user_id
```

Comment:
this is not valid in python3 -- print needs parens (and works then in both) -- probably should remove any print statements outside the TEST_DATA test.

Also, those look like real IDs -- so maybe add a letter or something to the test ids to make sure it's impossible?